stored_requests/events/http/http.go

```diff
@@ -94,10 +95,33 @@ func (e *HTTPEvents) refresh(ticker <-chan time.Time) {
 		select {
 		case thisTime := <-ticker:
 			thisTimeInUTC := thisTime.UTC()
-			thisEndpoint := e.Endpoint + "?last-modified=" + e.lastUpdate.Format(time.RFC3339)
+
+			// Parse the endpoint url defined
```

Comment:
Consider extracting the new code to a method, something like:

```
url.RawQuery = buildUpdateURL(e.Endpoint, e.lastUpdate);
```

Reply:
If a new stand alone function is required it might as well be more generic then that and pass a structure with the query params to update.  This function should be moved into some kind of utils file so that other parts of the project can possibly utilize it in the future.  Currently that is 
beyond my knowledge in GoLang at this point or available time to learn. 

I would imagine something like
```golang
endpointUrl.RawQuery = updateURLQuery(url, struct_of_query_params)
```

Reply:
Gotcha. Mind if we refactor this later on?

Reply:
If you are asking me that is completely up to you guys.  If you want to change the code now or later to make it better and more testable that is fine.  I just wanted to get it working so the PBS would stop making invalid url calls when trying to do amp updates.

Reply:
I'd like to get the fix in ASAP as well. Wanted to first make sure you won't be offended if you see us refactor it a bit later on.

Reply:
Not a problem. I am not at all good at GoLang so I am sure it can be done better so I expect it will be changed or even removed at a later time.